content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -249,6 +249,63 @@ the server will use to verify its clients. Create the secret `istio-ingressgatew
 
     This time the server performed client authentication successfully and you received the pretty teapot drawing again.
 
+## Configure end-user authentication on ingress gateway
+
+Ingress gateway supports end-user authentication by setting up JWT authentication policy on `istio-ingressgateway`, unauthenticated requests will be rejected at ingress gateway level and won't be able to access services inside the mesh. This section shows how to use authentication policy to setup end-user authentication on `istio-ingressgateway`.
+
+1. Perform the steps in the [Before you begin](/docs/tasks/traffic-management/ingress#before-you-begin), verify the setup by running curl command from before will return with 200.
+
+    {{< text bash >}}
+    $ curl -I -HHost:httpbin.example.com http://$INGRESS_HOST:$INGRESS_PORT/status/200
+    HTTP/1.1 200 OK
+    {{< /text >}}
+
+1. Add a policy that requires end-user JWT on `istio-ingressgateway`.
+
+    {{< text bash >}}
+    $ cat <<EOF | kubectl apply -f -
+    apiVersion: "authentication.istio.io/v1alpha1"
+    kind: "Policy"
+    metadata:
+      name: "ingressgateway"
+      namespace: istio-system
+    spec:
+      targets:
+      - name: istio-ingressgateway
+      origins:
+      - jwt:
+          issuer: "testing@secure.istio.io"
+          jwksUri: "{{< github_file >}}/security/tools/jwt/samples/jwks.json"
+      principalBinding: USE_ORIGIN
+    EOF
+    {{< /text >}}
+
+1. The same curl command from before will return with 401 error code, as a result of server is expecting JWT but none was provided:
```

Comment:
Do not reference a previous curl command vaguely. This document has several curl commands and the reader would just get confused. Particularly, since you have the command right below. I suggest:
1. Run the following curl command and note the return value of 401. Istio returns this error code value because the server expects a JWT but the ingress gateway did not provide one:

Reply:
updated.